tools/continuous-ingestor/timestream_sample_continuous_data_ingestor_application.py

```diff
@@ -325,6 +459,9 @@ def run(self):
 
 def ingestRecords(tsClient, dimensionsMetrics, dimensionsEvents, args):
     numThreads = args.concurrency
+    if (numThreads > len(dimensionsMetrics)):
```

Comment:
Un-Pythonic to have parentheses in `if`. Please remove here and in all changes.

Reply:
Fixed this in 12 places.